### ISSUE TITLE
Mention RMarkdown?

### ISSUE THREAD

**Issue by emilydolson** [swcarpentry/modern-scientific-authoring#26]
The ability to have your document automatically update when you subtly change your data or analysis seems like a pretty big selling point and has the added benefit of making your research more easily reproducible, particularly now that RMarkdown handles so many other languages (I think a lot of people discount RMarkdown because they don't know about the "engine" setting on code chunks). Or maybe this is too advanced.

**Comment by plantarum** [comment]
I agree with this idea, sort of: combining code a text in the same document is a game-changing improvement in manuscript prep.  In fact, I'm not sure I would recommend switching to Markdown or LaTeX _without_ using RMarkdown (or .Rnw in LaTeX).

My only reluctance here is noting that this is the **third** variant format that has been suggested for addition. Mark-up for manuscripts suffers from serious and ongoing fragmentation. Compared to WYSIWYG, where, for better or worse, we have a single reference implementation, it doesn't help our 'cause' that there are multiple flavours of two major markdown approaches, plus Rst, org-mode, html...
